projects/storefrontapp-e2e-cypress/cypress/e2e/regression/asm/create-customer.e2e-flaky.cy.ts

```diff
@@ -120,7 +118,9 @@ context('Assisted Service Module', () => {
           'Saved items'
         );
       });
+    });
 
+    it('vertify login to Spartacus (CXSPA-1594)', () => {
```

Comment:
For this scenario, I don't think that split into two cases, otherwise, it's not a complete scenario. You can see the [scenario](https://jira.tools.sap/browse/CXSPA-1594) form Linda's comment:
```
Cypress scenario: 
Asagent access SPA ASM then click top-right link "create customer"
Filling customer info then save
Customer is created within success message and emulated
Go to backoffice and set password for customer (Can API to cover?)
Back to SPA then new customer should be login in
```